### ISSUE TITLE
Order Panel filter filters for status which doesn't exist

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2881]
### Expected behavior
If I filter for "Captured" I should see orders that are captured but not shipped

### Actual Behavior
No orders ever qualify for this filter as I don't think this is a workflow state.

### Steps to Reproduce the Behavior
1. Put in an order
1. Go to order panel
1. Capture that order
1. Filter for "Captured"
1. Observe that no orders show up

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: esther-fix-issue-2821
Docker: 17.06.2-ce
```

**Comment by aaronjudd** [comment]
for `marketplace` v1.5.0 we should hide/disable/remove UI functionality that doesn't exist.

**Comment by Akarshit** [comment]
Will try to look into this.

**Comment by Akarshit** [comment]
@aaronjudd We make queries that use a combination of `billingStatus` and `workflow.status` to implement these filters.
From what I could see, only the query for `captured` filter was incorrect. So, I think nothing needs to be removed. 
Though one possible issue is even if the user selects `Approved` or `Captured`, though we are showing the correct results, we show the status as 'Processing` in the `Status` column.

**Comment by aaronjudd** [comment]
@Akarshit 👍  makes sense, thanks.  @spencern will review and we should get this into the next release.

**Comment by spencern** [comment]
![image](https://user-images.githubusercontent.com/1203639/30894389-c1512c02-a300-11e7-936d-ef24203a456e.png)
Captured works now. 👍 
Thanks for the fix @Akarshit!

**Comment by spencern** [comment]
Resolved by #2924
